index.html

```diff
@@ -25,15 +25,140 @@
+                    </div>
+                    <div class="flex-item6"><button>See project</button></div>
+                 </div>
+                 <div class="grid-item">
+                    <div class="flex-item1"><img src="./images/Snapshoot4.png" alt="my tonic project"></div>
+                    <div class="flex-item2"><h1>Multi-Post<br> Stories</h1></div>
+                    <div class="flex-item3"> 
+                        <ul>
+                            <li><a href="#" class="canopy">CANOPY</a></li>
+                            <li><a href="#"><img src="./images/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">Back End Dev</a></li>
+                            <li><a href="#"><img src="./images/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">2015</a></li>
+                        </ul>
+                    </div>
+                    <div class="flex-item4"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                    <div class="flex-item5">
+                        <ul>
+                            <li class="html"><a href="#">html</a></li>
+                            <li class="css"><a href="#">css</a></li>
+                            <li class="javaScript"><a href="#">javaScript</a></li>
+                        </ul>
+                    </div>
+                    <div class="flex-item6"><button>See project</button></div>
+                 </div>
+           </div>
+        </section>
+        <section class="about-myself">
+            <h1>About <br> Myself</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <P class="connect">LET'S CONNECT</P>
+            <ul class="contact-icon">
+            <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+            </ul>  
+            <button>Get my resume</button>     
+            <section class="expertise">
+                <ul class="language">
+                    <li><p>Language</p></li>
+                    <li><i class="fa fa-chevron-down" aria-hidden="true"></i></li>
+                </ul>
+                    <ul class="programming">
+                        <li><img src="./images/Frame9.png" alt="JavaScript"></li>
+                        <li><img src="./images/Frame10.png" alt="HTML"></li>
+                        <li><img src="./images/Frame11.png" alt="CSS"></li>
+                    </ul>
```

Comment:
- [ ] I request that you style the technology category lists instead of selecting them as an image. This is because styling does not wither an element as seen in your design.

| Your design | Figma design |
| --- | --- |
| <img width="303" alt="ht" src="https://user-images.githubusercontent.com/4247740/143521171-d987bd92-c119-4970-9d59-eaeadf099ee7.PNG"> | <img width="242" alt="ht2" src="https://user-images.githubusercontent.com/4247740/143521185-cf800993-13f7-4734-88e6-16bf6bc3a879.PNG"> |

Reply:
I have changed the technology categories into ul list.